docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+# Cellxgene Curation Tutorial
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png) --->
+
+<img align="right" src="https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png">
+
+
+<!--- ## Overview --->
+<!--- purpose: Motivate submission to the data portal give context --->
+
+[Cellxgene's publishing platform](https://cellxgene.cziscience.com/) and [interactive single cell data explorer](https://github.com/chanzuckerberg/cellxgene) is a system which is optimized for access, exploration and reuse of single cell data. In order to achieve these goals, the cellxgene platform currently accepts curated [AnnData](https://anndata.readthedocs.io/en/latest/#) objects adhering to a [succinct data schema](corpora_schema.md). Adherence to a standardized data schema allows for efficient navigation and integration of the growing number of single cell datasets that are becoming available. In this tutorial, you will learn the essential information for curating a single cell dataset using CZI's curation tools and uploading to the data portal. Hosting your data on the cellxgene portal will offer the following benefits:
```

Comment:
_Cellxgene's publishing platform](https://cellxgene.cziscience.com/) and [interactive single cell data explorer](https://github.com/chanzuckerberg/cellxgene) ~is a system which is~ are optimized for access, exploration and reuse of single cell data. ~In order t~ To achieve these goals, the cellxgene platform currently accepts curated ..._

What is meant by "efficient navigation" ? For example, the schema is unrelated to the collection model.

_integration of the growing number of single cell datasets ~~that are becoming available~~_

_Hosting your data on the cellxgene portal ~will offer~ offers the following benefits:_

Reply:
"efficient navigation" refers to the ability to filter datasets by assay, organism tissue, etc. - different than exploration 

phrasing suggestions accepted and changed